### PR TITLE
OPSEXP-2445 Prepare for 8.0.0-M.1 release

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2445

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ The table below shows the exact version of ACS deployed with each chart version/
 | [7.0.1][7.0.1]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
 | [7.0.2][7.0.2]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
 | [7.0.3][7.0.3]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
+| [7.1.0-M.1][7.1.0-M.1] | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
```
Comment (alxgomz) [inline README.md:92]:
```suggestion
| [7.1.0-M.1][7.1.0-M.1] | 23.2.0-M1  | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
```

File: helm/alfresco-content-services/Chart.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # https://www.alfresco.com/platform/content-services-ecm/trial/download
 apiVersion: v2
 name: alfresco-content-services
-version: 7.0.3
+version: 7.1.0-M.1
 appVersion: 23.1.1
```
Comment (alxgomz) [inline helm/alfresco-content-services/Chart.yaml:8]:
```suggestion
appVersion: 23.2.0-M1
```

**Review by alxgomz — CHANGES_REQUESTED** [review]
I’m wondering if ATS, ASE & ADW bump are not early Eridanus work vs Draco stuff.
There was no release for these components in draco

File: docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -159,7 +159,7 @@ services:
       APP_CONFIG_PLUGIN_MICROSOFT_ONLINE: "false"
       APP_BASE_SHARE_URL: "http://localhost:8080/workspace/#/preview/s"
   control-center:
-    image: quay.io/alfresco/alfresco-control-center:8.3.0
+    image: quay.io/alfresco/alfresco-control-center:4.3.0
```
Comment (gionn) [inline docker-compose/docker-compose.yml:162]:
```suggestion
    image: quay.io/alfresco/alfresco-control-center:8.3.0
```

File: test/enterprise-integration-test-values.yaml [hunk 4]
```diff
@@ -125,3 +125,14 @@ alfresco-sync-service:
         limits:
           cpu: "2"
           memory: "1Gi"
+global:
+  ai:
+    enabled: true
```
Comment (alxgomz) [inline test/enterprise-integration-test-values.yaml:130]: to remove

**Review by alxgomz — CHANGES_REQUESTED** [review]

**Review by alxgomz — DISMISSED** [review]

**Review by alxgomz — APPROVED** [review]

**Pull request merged**